### DISCUSSION TITLE
Docs example deep links

### DISCUSSION THREAD

**Discussion by br41nslug** [directus/directus#19902] [Feature Requests]
### Summary

When helping folks in Discord i often have to share docs links to specific examples. The issue is that examples themselves are not directly linkable only bigger headings are so i often have to look up for the first linkable heading and tell the receiver to scroll down and toggle to the required example.

### Motivation

When helping folks in Discord i often have to share docs links to specific examples. And having to go through extra steps and add extra instructions costs time that could have been spent on other things.

### Detailed Design

For example: on the "global parameters" page if i want to link to the fields example block i have to link to the "Fields" title and scroll down to example

https://docs.directus.io/reference/query.html#fields

If each of these tabs had their own anchor to jump to sharing a link to one of the following:
- https://docs.directus.io/reference/query.html#fields-example-graphql
- https://docs.directus.io/reference/query.html#fields-example-sdk

Could directly scroll to the example block and even directly show the wanted example language.

### Must Have:
- An anchor point per example to scroll to with a direct link

### Should Have:
- An anchor point to jump to a specific tech tab in the example

### Drawbacks

- Will probably take some work to apply to all code examples

### Alternatives

Sharing screenshots of the correct code block in Discord which is sub-optimal.

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
